target/linux/mvebu/image/turris-omnia.bootscript

```diff
@@ -0,0 +1,23 @@
+# Standard Boot-Script
+# use only well-known variable names provided by U-Boot Distro boot
+# This script assumes that there is a boot partition,
+# and that the root partition is always the next one.
+
+# rootfs is always on the next partition
+setexpr openwrt_rootpart ${distro_bootpart} + 1
+
+# figure out partition uuid to pass to the kernel as root=
+part uuid ${devtype} ${devnum}:${openwrt_rootpart} uuid || part uuid ${devtype} ${devnum}:${distro_bootpart} uuid
+
+# generate bootargs (rootfs)
+setenv bootargs earlyprintk root=PARTUUID=${uuid} rootfstype=auto rootwait
+
+# add console= option to bootargs, if any
+if test -n "${console}"; then
```

Comment:
I assume

[ -n "${console}" ] && setenv bootargs ${bootargs} console=${console}

also works.

Reply:
I don't see an advantage in making this different from clearfog.bootscript in the same directory.